zfit/models/kde.py

```diff
@@ -428,7 +428,7 @@ def _convert_init_data_weights_size(self, data, weights, padding):
         if callable(padding):
             data, weights = padding(data=data, weights=weights)
         elif padding is not False:
-            data, weights = padreflect_data_weights_1dim(data, weights=weights, mode=padding)
+            data, weights = padreflect_data_weights_1dim(data, weights=weights, mode=padding, limits=None)
```

Comment:
```suggestion
            data, weights = padreflect_data_weights_1dim(data, weights=weights, mode=padding, limits=limits)
```
this should be limits, right?